pkgs/by-name/cm/cmctl/package.nix

```diff
@@ -35,36 +31,25 @@ buildGoModule rec {
     installShellFiles
   ];
 
-  # Trusted by this computer: no: x509: “cert-manager” certificate is not trusted
-  doCheck = !stdenv.hostPlatform.isDarwin;
+  checkPhase = ''
+    go test --race $(go list ./... | grep -v /test/)
+  '';
 
-  postInstall =
-    ''
-      mv $out/bin/ctl $out/bin/cmctl
-    ''
-    + lib.optionalString (stdenv.buildPlatform.canExecute stdenv.hostPlatform) ''
-      installShellCompletion --cmd cmctl \
+  postInstall = lib.optionalString (stdenv.buildPlatform.canExecute stdenv.hostPlatform) ''
+    installShellCompletion --cmd cmctl \
         --bash <($out/bin/cmctl completion bash) \
         --fish <($out/bin/cmctl completion fish) \
         --zsh <($out/bin/cmctl completion zsh)
-    '';
-
-  passthru.updateScript = nix-update-script { };
+  '';
 
   meta = with lib; {
-    description = "CLI tool for managing cert-manager service on Kubernetes clusters";
+    description = "`cmctl` is the command line utility that makes cert-manager'ing easier.";
```

Comment:
Please make sure the description follows the style described in the manual https://nixos.org/manual/nixpkgs/unstable/#var-meta-description

Reply:
Fixed